lisp/ess-inf.el

```diff
@@ -1959,6 +1959,7 @@ for `ess-eval-region'."
     (define-key map "\M-?"     'ess-list-object-completions)
     (define-key map "\C-c\C-k" 'ess-request-a-process)
     (define-key map ","        'ess-smart-comma)
+    (define-key map (kbd "M-_") 'ess-insert-S-assigns)
```

Comment:
I din't mean this literally. Just as an example.

Reply:
Ah, sorry for the misunderstanding. I'll remove that commit.